### ISSUE TITLE
Consider left-aligning the event page banner

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#186]
## Description

Banner's content should be left-aligned (not center-aligned) to match the figma design 🙇 it will help a bit with accessibility too for folks using magnifiers who may not anticipate that a non-landing page has a centered banner.

<img width="1508" alt="Screen Shot 2022-09-29 at 4 48 53 PM" src="https://user-images.githubusercontent.com/14154792/193139097-4254f1cd-9fe4-447a-800c-01cc73437356.png">

## Success criteria

- [ ] Event page banner is left-aligned
